src/App.tsx

```diff
@@ -1,61 +1,112 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function findUserById(id: number) {
+  return usersFromServer.find(user => user.id === id) || null;
+}
+
+const todoList = todosFromServer.map(todo => (
```

Comment:
Consider moving the logic for creating `todoList` to a separate function. It will make the code more modular and reusable. 🧩